tests/test_uwsgi_plugin.py

```diff
@@ -0,0 +1,62 @@
+from testing.sandbox import install_fake_uwsgi
+install_fake_uwsgi()
```

Comment:
based on  your test ordering, this'll leak into the other tests

I'd suggest instead doing these tests in a subprocess (similar to how the clog config tests work)